plugins/juju/juju.plugin.zsh

```diff
@@ -163,10 +163,40 @@ jreld() {
   juju run "relation-get -r $relid - $2" --unit $2/$3
 }
 
+# Return Juju current controller
+jcontroller() {
+  local controller="$(grep "current-controller"  ~/.local/share/juju/controllers.yaml  | awk '{print $2}')"
+  if [[ -z "$controller" ]]; then
+    return 1
+  fi
+
+  echo $controller
+  return 0
+}
+
+# Return Juju current model
+jmodel() {
+  local yqbin="$(whereis yq | awk '{print $2}')"
+
+  if [[ -z "$yqbin" ]]; then
+    echo "--"
+    return 1
+  fi
+
+  local model="$(yq e ".controllers.$(jcontroller).current-model" < ~/.local/share/juju/models.yaml | awk -F "/" '{print $2}')"
```

Comment:
You can use `| cut -d/ -f2` instead of that `awk` call, it's a bit faster.

Reply:
Done